classes/travel/ObjectiveLocation.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class ObjectiveLocation {
```

Comment:
What do you think about naming this `MapObjectiveLocation` just for slightly more clarity?

Reply:
Changed in commit.